src/clj/game/cards/agendas.clj

```diff
@@ -515,11 +515,12 @@
 
    "Personality Profiles"
    (let [pp {:req (req (pos? (count (:hand runner))))
-             :msg "force the Runner to trash 1 card from their Grip at random"
-             :effect (effect (trash (first (shuffle (:hand runner)))))}]
+             :effect (effect (trash (first (shuffle (:hand runner)))))
+             :msg (msg "force the Runner to trash 1 card from their Grip at random. Trashes: "
+                       (:title (first (:discard runner))))}]
```

Comment:
Thanks for the review.  Some clojure style i need to learn... as will move toward (eg. pos? zero?)

I tried putting the trashed card names in the main text - but to me it is confusing.  Are you ok if i put brackets around them?  

For example on Ramujan this happens:

wozzit uses Ramujan-reliant 550 BMI to trash 1 cards from their Stack Rolodex and prevent 1 damage.

To me the word Rolodex looks weird there - unless I surround with brackets or maybe a - either side